targets/ramips-mt7620

```diff
@@ -7,3 +7,10 @@ factory
 
 device gl-mt750 gl-mt750
 factory
+
+# TP-Link
+# BROKEN: only 2.4GHz wifi working
```

Comment:
please add this mark at the end of the "device" line as it's done for the other devices